features/s3/postObject.feature

```diff
@@ -1,28 +1,14 @@
 @s3 @integ
 Feature: POST object uploads
 
-    Scenario: Upload an object via POST object with Signature V2
```

Comment:
I'm pretty sure I'm the one who deleted this since you can't make new S3 buckets that accept V2- for legacy customers, we should add a backlog item to ticket S3 to allowlist an account for this and run it there so we still have this tested for customers using legacy buckets.